.travis.yml

```diff
@@ -6,7 +6,7 @@ services:
   - docker
 
 go:
-  - "1.13"  # Keep in sync with Makefile
+  - "1.13.6"  # Keep in sync with Makefile
```

Comment:
Since you're here, and you caught a sync issue in the README.md, can you update the comment here and in Makefile to also update README.md?